src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj

```diff
@@ -3,12 +3,14 @@
   <PropertyGroup>
     <Description>Microsoft Azure Monitor Library</Description>
     <AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
-    <Version>0.16.1-preview</Version>
+    <Version>0.17.0-preview</Version>
     <PackageId>Microsoft.Azure.Management.Monitor</PackageId>
     <PackageTags>Management.Monitor;Management.Monitoring;metrics;alerts;autoscale;activityLogs;events;operations;logs</PackageTags>
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
     <RootNamespace>Microsoft.Azure.Management.Monitor</RootNamespace>
+    <DelaySign>false</DelaySign>
```

Comment:
@vcanaa these properties are set for the entire repo, so no need to set it again.
Remove delaySign and SignAssembly properties